python/ray/_private/runtime_env/context.py

```diff
@@ -69,7 +68,7 @@ def exec_worker(self, passthrough_args: List[str], language: Language):
         command_str = " && ".join(self.command_prefix + [exec_command])
         logger.debug(f"Exec'ing worker with command: {command_str}")
         if sys.platform == "win32":
-            os.system(command_str)
+            subprocess.check_call([executable, *passthrough_args])
```

Comment:
Did you try `check_call`? I think some of the processes exit with `1` as an exit code, so in the other PR I used `subprocess.run` instead. I think the error checking is done by other ray mechanisms: checking that the worker registers back with the raylet and emits proper heartbeats.

Reply:
I should probably switch to subprocess.run yeah, according to the python docs it's better